graylog2-web-interface/src/components/event-definitions/event-definition-types/FilterPreview.jsx

```diff
@@ -54,15 +55,30 @@ class FilterPreview extends React.Component {
     const { isFetchingData, searchResult, errors } = this.props;
 
     return (
-      <div className={styles.filterPreview}>
-        <h3>Filter preview</h3>
-        {errors.length > 0 ? (
-          <p className="text-danger">{errors[0].description}</p>
-        ) : (
-          isFetchingData ? <Spinner text="Loading filter preview..." /> : this.renderSearchResult(searchResult)
-        )
-        }
-      </div>
+      <React.Fragment>
+        <HelpPanel collapsible
+                   title="How many Events will Filter & Aggregation create?">
+          <p>
+            The Filter & Aggregation Condition will generate different number of Events, depending on how it is
+            configured:
+          </p>
+          <ul>
+            <li><b>Filter:</b>&emsp;One Event per message matching the filter</li>
+            <li><b>Aggregation without groups:</b>&emsp;One Event every time the Event Definition is executed</li>
```

Comment:
I think "one event every time the event definition is executed" is a bit misleading. It's more like: "one event every time the condition matches the result of the message aggregation" (same for the "per group" case)

Reply:
I tried to make it more accurate now 👍